proto/services/task_svc/v1/task_template_svc.proto

```diff
@@ -24,6 +24,35 @@ message CreateTaskTemplateResponse {
   string id = 1;
 }
 
+
+///
+/// GetAllTaskTemplates
+///
+
+message GetAllTaskTemplatesRequest {
+}
+
+message GetAllTaskTemplatesResponse {
+  repeated TaskTemplate templates = 1;
+
+  message TaskTemplate {
+    string id = 1;
+    string name = 2;
+    string description = 3;
+    bool is_public = 4;
+    optional string user_id = 6;
```

Comment:
the name "user_id" is bad. what kind of user? creator / owner? assignee? If I fear this is consistent (in that case it should stay like it is) though

Reply:
we should discuss this whether owner or creator is more appropriate

Reply:
but since it is used in all other PR's of #200, we should use a seperate issue

Reply:
see #226